### DISCUSSION TITLE
Simplify Release Process

### DISCUSSION THREAD

**Discussion by bevans2000** [mars-sim/mars-sim#1069] [General]
The release process is quite labour intensive. We should simplify it to make it easier to run and understand for an end user. It involves creating 4 separate files as Artefacts and various change log files. I suggest we drop the DEB file and go to a single ZIP artefact that contains both JAR files and support both OS.

- jars/mars-sim_3.6.2_console.jar
- jars/mars-sim_3.6.2_swing.jar
- bin/mars-sim-ui.bat
- bin/mars-sim-ui.sh
- bin/mars-sim-headless.bat
- bin/mars-simheadlessi.sh
- readme.txt
- changelog
-conf/jvm.options

This is common pattern in lots of Java based application, Apache JMeter is a good example. This will simplify the wiki pages as well since we can describe a single execution & installation model. 
To create this we have a new Maven module that will just do the packaging.

@mokun Opinion?

**Comment by mokun** [comment]
Sure. The jar version is good enough as not a whole lot of people are downloading the deb versions anyway. Let's do that.

**Reply by bevans2000** [reply]
Covered by #1072
